samples/net/sockets/http_client/prj.conf

```diff
@@ -45,3 +45,5 @@ CONFIG_LOG_MODE_IMMEDIATE=y
 CONFIG_NET_LOG=y
```

Comment:
Typo in commit message of `samples: net: http_client/server: addtl logging boilerplate`
`s/addtl`, not sure what you wanted to say here

Reply:
Fixed.